### ISSUE TITLE
ROM's autoplay

### ISSUE THREAD

**Issue by CeitDe** [NovaSquirrel/Mesen-X#92]
Hi I installed Mesen on Windows 11 and went to play ROMs, each ROM I played it would only go into Demo view.  The ROM would not run the game. 

Is there a known issue?

**Comment by NovaSquirrel** [comment]
What do you mean by "Demo view"? Have you set up the controls and tried to press Start?

**Comment by CeitDe** [comment]
Yes, I have set up the controls and tried to press Start, but instead of going into the controls, it automatically starts to runs a simulation of the game - The controls nor the keyboard commands work. No matter what game it is.
